src/main/kotlin/no/nav/familie/ks/sak/kjerne/beregning/domene/Sats.kt

```diff
@@ -0,0 +1,45 @@
+package no.nav.familie.ks.sak.kjerne.beregning.domene
+
+import no.nav.familie.ks.sak.common.util.toYearMonth
+import java.math.BigDecimal
+import java.math.RoundingMode
+import java.time.LocalDate
+import java.time.YearMonth
+
+data class Sats(
+    val type: SatsType,
+    val beløp: Int,
+    val gyldigFom: LocalDate = LocalDate.MIN,
+    val gyldigTom: LocalDate = LocalDate.MAX
+)
+
+enum class SatsType(val beskrivelse: String) {
+    KS("Ordinær kontantstøtte"),
+}
+
+data class SatsPeriode(val sats: Int, val fom: YearMonth, val tom: YearMonth, val prosent: BigDecimal)
+
+private val sats = Sats(type = SatsType.KS, beløp = 7500, gyldigFom = LocalDate.MIN, gyldigTom = LocalDate.MAX)
+
+fun maksBeløp() = sats.beløp
+
+fun hentGyldigSatsFor(antallTimer: BigDecimal?, erDeltBosted: Boolean, stønadFom: YearMonth, stønadTom: YearMonth): SatsPeriode {
+    val prosent = when {
+        erDeltBosted -> 50
+        antallTimer == null -> 100
+        antallTimer in BigDecimal(0)..BigDecimal(8) -> 80
+        antallTimer in BigDecimal(9)..BigDecimal(16) -> 60
+        antallTimer in BigDecimal(17)..BigDecimal(24) -> 40
+        antallTimer in BigDecimal(25)..BigDecimal(32) -> 20
+        else -> 0
```

Comment:
Denne må visst skrives om. Den ekskluderer tall mellom 8 og 9, 16 og 17 og mellom 24 og 25.

Reply:
Fordi maks antall desimaler er 2, kan den muligens skrives om til:
```suggestion
    val prosent = when {
        erDeltBosted -> 50
        antallTimer == null -> 100
        antallTimer in BigDecimal(0)..BigDecimal(8.99) -> 80
        antallTimer in BigDecimal(9)..BigDecimal(16.99) -> 60
        antallTimer in BigDecimal(17)..BigDecimal(24.99) -> 40
        antallTimer in BigDecimal(25)..BigDecimal(32.99) -> 20
        else -> 0
    }
```

Reply:
fikset det